src/ResourceManager/Common/Commands.ResourceManager.Common/Generated/SubscriptionsOperationsExtensions.cs

```diff
@@ -167,9 +168,9 @@ public static IPage<Subscription> ListNext(this ISubscriptionsOperations operati
             /// <returns>
             /// PageEnumerable object containing all of the subscriptions for the given tenant used in the client.
             /// </returns>
-            public static PageEnumerable ListAll(this ISubscriptionClient client)
```

Comment:
I don't think we can remove these methods - we need to only make addivie changes to the common code

Reply:
@markcowl reverted `PageEnumerable` and `PageEnumerator` to their original form (only `SubscriptionClient` is handled) and moved all the new functionality into `GenericPageEnumerable` and `GenericPageEnumerator`